docs/internal/GITHUB_BRANCH_PROTECTION_RUNBOOK.md

```
@@ -130,13 +130,13 @@ Use this sequence:
 1. Inspect what branch protection currently requires:
 
 ```bash
-GITHUB_TOKEN="$GITHUB_PAT" gh api repos/pdcarlson/Frapp/branches/main/protection
+gh api repos/pdcarlson/Frapp/branches/main/protection
 ```
 
 2. Inspect what the PR actually reported:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)